src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.2" />
```

Comment:
Is there a better way to update this automatically?

Reply:
I am gonna create an issue to track this work.

Reply:
We could use Versions.props and DARC. We have an issue for this that we need to figure out which dependencies from dotnet/runtime we need to have setup as subscriptions from the repo itself and which should be live dependencies.